CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/CardExtensions.cs

```diff
@@ -84,5 +87,55 @@ public static Attachment ToAttachment(this FileConsentCard card)
                 Name = card.Name,
             };
         }
+
+        /// <summary>
+        /// Creates a new attachment from AdaptiveCard.
+        /// </summary>
+        /// <param name="card"> The instance of AdaptiveCard.</param>
+        /// <returns> The generated attachment.</returns>
+        public static Attachment ToAttachment(this AdaptiveCards.AdaptiveCard card)
+        {
+            return new Attachment
+            {
+                Content = card,
+                ContentType = AdaptiveCards.AdaptiveCard.ContentType
+            };
+        }
+
+        /// <summary>
+        /// Creates a new attachment from AdaptiveCardParseResult.
+        /// </summary>
+        /// <param name="cardParsedResult"> The instance of AdaptiveCardParseResult that represents results parsed from JSON string.</param>
+        /// <returns> The generated attachment.</returns>
+        public static Attachment ToAttachment(this AdaptiveCards.AdaptiveCardParseResult cardParsedResult)
+        {
+            return cardParsedResult.Card.ToAttachment();
+        }
+
+        /// <summary>
+        /// Wrap BotBuilder action into AdaptiveCard.
+        /// </summary>
+        /// <param name="action"> The instance of adaptive card.</param>
+        /// <param name="targetAction"> Target action to be adapted.</param>
+        public static void RepresentAsBotBuilderAction(this AdaptiveCards.AdaptiveSubmitAction action, CardAction targetAction)
+        {
+            var wrappedAction = new CardAction
+            {
+                Type = targetAction.Type,
+                Value = targetAction.Value,
```

Comment:
You can do JObject.FromObject(targetAction).ToObject<CardAction>()

Reply:
Or infact just do JObject.FromObject and then ToString() below

Reply:
@RamjotSingh  but i don't want to copy "Title"